src/view/film-card.js

```diff
@@ -30,25 +30,58 @@ const createFilmCardTemplate = (card) => {
 class FilmCard extends AbstractView{
   constructor(card){
     super();
-    this._card = card;
+    this.card = card;
```

Comment:
```suggestion
    this._card = card;
```
это приватное свойство. к нему нельзя обращаться из вне.